drivers/gpio/gpio_xlnx_axi.c

```diff
@@ -247,76 +242,111 @@ static const struct gpio_driver_api gpio_xlnx_axi_driver_api = {
 	.get_pending_int = gpio_xlnx_axi_get_pending_int,
 };
 
-#define GPIO_XLNX_AXI_GPIO2_HAS_COMPAT_STATUS_OKAY(n)			\
-	UTIL_AND(							\
-		DT_NODE_HAS_COMPAT(DT_CHILD(DT_DRV_INST(n), gpio2),	\
-				   xlnx_xps_gpio_1_00_a_gpio2),		\
-		DT_NODE_HAS_STATUS(DT_CHILD(DT_DRV_INST(n), gpio2),	\
-				   okay)				\
+#define GPIO_XLNX_AXI_GPIO2_HAS_COMPAT_STATUS_OKAY(n)		    \
+	UTIL_AND(						    \
+		DT_NODE_HAS_COMPAT(DT_CHILD(DT_DRV_INST(n), gpio2), \
+				   xlnx_xps_gpio_1_00_a_gpio2),	    \
+		DT_NODE_HAS_STATUS(DT_CHILD(DT_DRV_INST(n), gpio2), \
+				   okay)			    \
 		)
```

Comment:
Please avoid unrelated whitespace changes. It makes the PR much harder to review.

Reply:
The whitespace change was the result of running uncrustify on the file before commiting.